packages/runtime-core/src/directives.ts

```diff
@@ -78,7 +78,7 @@ export type DirectiveArguments = Array<
   | [Directive | undefined]
   | [Directive | undefined, any]
   | [Directive | undefined, any, string]
-  | [Directive | undefined, any, string, DirectiveModifiers]
+  | [Directive | undefined, any, string | undefined, DirectiveModifiers]
```

Comment:
line 80 also needs this change.

Reply:
If it is `[_vModelText, $setup.msg, void 0]`, would it be correct to write it as follows?

```
[_vModelText, $setup.msg]
```

This would correspond to the type on line 79.

Reply:
uh~ yes.